index.html

```diff
@@ -193,55 +203,55 @@ <h3 class="categories-link-text">Квадрокоптеры</h3>
       <h2 class="visually-hidden">Условия оплаты и гарантии</h2>
       <ol class="proposals-pagination">
         <li class="proposals-pagination-item">
-          <button class="proposals-pagination-button proposals-pagination-button-current" type="button">
+          <button class="button proposals-pagination-button proposals-pagination-button-current" type="button">
             Доставка
           </button>
         </li>
         <li class="proposals-pagination-item">
-          <button class="proposals-pagination-button" type="button">
+          <button class="button proposals-pagination-button" type="button">
             Гарантия
           </button>
         </li>
         <li class="proposals-pagination-item">
-          <button class="proposals-pagination-button" type="button">
+          <button class="button proposals-pagination-button" type="button">
             Кредит
           </button>
         </li>
       </ol>
       <section class="proposals-item">
-        <h3 class="proposals-title">Доставка</h3>
-        <p class="proposals-text">
+        <h3 class=" main-title proposals-title">Доставка</h3>
+        <p class="plain-text proposals-text">
           Мы с удовольствием доставим ваш товар прямо к вашему подъезду
           совершенно бесплатно! Ведь мы неплохо заработаем, поднимая его на
           ваш этаж.
         </p>
       </section>
       <section class="proposals-item">
-        <h3 class="proposals-title">Гарантия</h3>
-        <p class="proposals-text">
+        <h3 class="main-title proposals-title">Гарантия</h3>
+        <p class=" plain-text proposals-text">
```

Comment:
```suggestion
        <p class="plain-text proposals-text">
```
Лишний пробел после открывающей кавычки.